### ISSUE TITLE
Make humanity work on more stuff

### ISSUE THREAD

**Issue by doctor4t** [Ladysnake/Requiem#303]
![image](https://user-images.githubusercontent.com/25477005/117655446-6190f500-b197-11eb-98ac-b649c63fd686.png)

Also I think combining Humanity books still doesn't work?

**Comment by Pyrofab** [comment]
Humanity's max level should already be fixed in latest release

**Comment by Dual-Iron** [comment]
Yeah, I submitted this bug report late. I was using 1.6.3. Sorry.
